### ISSUE TITLE
[10.7.3] Layer Normalization

### ISSUE THREAD

**Issue by staycozyboy** [woowacampers/d2l-pytorch#19]
>  Layer normalization is the same as batch normalization except that the former normalizes across the feature dimension.
> 10.7.3

일단 위의 문장이 이해가 안됩니다.. 저도 따로 공부할 생각이지만, 혹시 이해하신 분 계신다면 간단한 설명 혹은 볼만한 레퍼런스 추천해주시면 감사하겠습니다 ㅎㅎ

```python
ln = nn.LayerNorm(2)
bn = nn.BatchNorm1d(2)
X = torch.tensor([[1, 2], [2, 3]], dtype=torch.float32)
# Compute mean and variance from `X` in the training mode
print('layer norm:', ln(X), '\nbatch norm:', bn(X))
```

추가로 위 코드의 의도(어떤 것을 보여주고자 하는지)도 명확하게 파악이 안됩니다 ㅠㅠ

**Comment by jiwon-ryu** [comment]
`ln(x)`와 `bn(x)`는 normalize된 x를 반환하는 것 같습니다.
계산을 해보면 다음과 같습니다.

1. 각 column 별로 정규화 할 경우
`mean` = 1.5 ,2.5, `std`=0.5, 0.5
-> [[-1 -1], [1, 1]]로 `bn(x)`와 같아집니다.
 
2. 각 row 별로 정규화 할 경우 
`mean` = 1.5 ,2.5, `std`=0.5, 0.5
-> [[-1, 1], [-1, 1]]로 `ln(x)`와 같아집니다.

**Comment by 1gyeol-KIM** [comment]
Batch Normalization과 Layer Normalization은 각각 Batch 차원, Feature 차원에서 정규화를 진행합니다. 이따 학습 콘텐츠에서 간략하게 설명해보겠습니다!
